src/commands/createStaticWebApp/addWorkspaceTelemetry.ts

```diff
@@ -14,7 +14,6 @@ export function addWorkspaceTelemetry(wizardContext: IStaticWebAppWizardContext)
     wizardContext.telemetry.properties.gotRemote = String(!!wizardContext.repoHtmlUrl);
 
     const localGit: git.SimpleGit = git(wizardContext.fsPath);
-    // tslint:disable-next-line:no-floating-promises
     localGit.status().then(() => {
```

Comment:
You don't need to put `void` here?

Reply:
ESLint doesn't complain about this - I think because we're making it synchronous with `.then()`